### PR TITLE
Update palo-alto-firewall.md - minor tweak to previously submitted PR

### PR DESCRIPTION
**Pull request by mvirga-sumo**
Update GlobalProtect Format

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
Docs Team - though this is approved, please allow @mvirga-sumo to merge when he's ready.

File: docs/cse/ingestion/ingestion-sources-for-cloud-siem/palo-alto-firewall.md
Comment (kimsauce) [inline docs/cse/ingestion/ingestion-sources-for-cloud-siem/palo-alto-firewall.md:104]: @mvirga-sumo we'll await your final signoff before merging

Comment (mvirga-sumo) [reply docs/cse/ingestion/ingestion-sources-for-cloud-siem/palo-alto-firewall.md:104]: @kimsauce  I have submitted the final PR that is comprehensive for this doc - that is PR #2875

**Comment by kimsauce** [comment]
closing in favor of https://github.com/SumoLogic/sumologic-documentation/pull/2875

**Pull request closed without merging**
